frontend/packages/integration-tests-cypress/reporter-config.json

```diff
@@ -9,6 +9,7 @@
     "reportFilename": "cypress_report",
     "overwrite": false,
     "html": false,
-    "json": true
+    "json": true,
+    "quiet" : true
```

Comment:
What output is this suppressing? Generally we want verbose output in CI for troubleshooting.

Reply:
This removes:
`[mochawesome] Report JSON saved to /home/dtaylor/repos/openshift/origin/src/github.com/openshift/console/frontend/gui_test_screenshots/cypress_report_004.json`

Reply:
Ok,  pulled unrelated changes.